servant-foreign/src/Servant/Foreign/Internal.hs

```diff
@@ -350,6 +350,20 @@ instance HasForeign lang ftype api
   foreignFor lang ftype Proxy req =
     foreignFor lang ftype (Proxy :: Proxy api) req
 
+instance HasForeign lang ftype api
```

Comment:
Here we ignore `Summary` and `Description`, but it would be super nice to adjust `servant-foreign` so that we could save descriptions to later translate them into comments.